Tukui/Libs/oUF_AuraTrack/oUF_AuraTrack.lua

```diff
@@ -428,7 +428,7 @@ local UpdateBar = function(self, unit, spellID, texture, id, expiration, duratio
 		return
 	end
 
-	local r, g, b = unpack(Tracker[spellID])
+	local r, g, b = unpack(AuraTrack.Tracker[spellID])
```

Comment:
```suggestion
	local r, g, b = unpack(Tracker[spellID])
```
The var Tracker is declared at line 3 and is set based if isRetail or not on line 24 or line 81 depending if true or not.  With ur isRetail changes, you should revert this part of the commit and just test with ur isRetail changes.